### DISCUSSION TITLE
Sending Files

### DISCUSSION THREAD

**Discussion by JeremypRTS** [dannagle/PacketSender#269] [Q&A] [answered]
I am trying to use a single file to send multiple packets of data through the command line. My text file looks similar to this:
packetsender -uaw 500 (IP) (Port) (Text)
packetsender -uaw 500 (IP) (Port) (Text)
packetsender -uaw 500 (IP) (Port) (Text)
packetsender -uaw 500 (IP) (Port) (Text)
packetsender -uaw 500 (IP) (Port) (Text)

I did this in hopes that Packet sender would open the file and send each message individually using the given delay, but it sends the whole file as one big packet. How should I proceed?

**Comment by dannagle** [comment, marked as answer]
The `-w 500` means "wait 500 ms for a response". It won't help you when using UDP. 

You could instead use a generic sleep command in between statements...

Example...
```
echo "sleep"
sleep 1
echo "done"
```

Also, if you are sending a file, you could use the `-f` parameter to send it directly.

```
packetsender -u -f "/path/to/file.txt" IP PORT
sleep 1
packetsender -u -f "/path/to/file.txt" IP PORT
sleep 1
packetsender -u -f "/path/to/file.txt" IP PORT
```

Keep in mind that UDP has a size limit that is typically around 1024 bytes.


Hope that helps.
